src/avram/needy_initializer_and_delete_methods.cr

```diff
@@ -91,22 +91,8 @@ module Avram::NeedyInitializerAndDeleteMethods
     %}
   end
 
-  macro generate_destroy(attribute_method_args, attribute_params, with_params, with_bang)
-    def self.delete{% if with_bang %}!{% end %}(*args, **named_args{% if !with_bang %}, &block{% end %})
-      {% verbatim do %}
-        {% raise <<-ERROR
-        DeleteOperations do not have a 'delete' method.
-
-        Try this...
-
-          ▸ Use 'destroy' to delete a record
-
-        ERROR
-        %}
-      {% end %}
-    end
```

Comment:
We should leave this in, but name it `destroy` with a message like "destroy has been renamed to delete". Maybe with the type name or something so it's easy to find which ones you have to replace?

Reply:
I added it in, unfortunately I had to move the compilation error because `@type` was not the inherited type and it was a mess of macro code in the backtrace.

This is something that I hope can be removed in the future so I don't think it's a big deal.